async/README.md

```diff
@@ -1,24 +1,46 @@
-### Use Async Feature
+# Async Sample
 
-Implement this interface
+### Background
+
+Dubbo-go provides not only synchronous invocation, but also asynchronous invocation on the consumer side. In order to
+use it, the client needs to implement the following interface to asynchronously receive the response from the service
+provider:
 
 ```golang
-//AsyncCallbackService callback interface for async
 type AsyncCallbackService interface {
-	CallBack(response CallbackResponse) // callback
+CallBack(response CallbackResponse)
```

Comment:
why delete the blank and the comment here?

Reply:
it's redundant, I think.

Reply:
move back.